diff --git a/target/linux/mediatek/dts/mt7987b-cudy-tr3600-v1.dts b/target/linux/mediatek/dts/mt7987b-cudy-tr3600-v1.dts
--- a/target/linux/mediatek/dts/mt7987b-cudy-tr3600-v1.dts
+++ b/target/linux/mediatek/dts/mt7987b-cudy-tr3600-v1.dts
@@ -82,7 +82,7 @@
 
 &fan {
 	fan-supply = <&fan_3p3v>;
-	pwms = <&pwm 1 50000 0>;
+	pwms = <&pwm 0 50000 0>;
 	status = "okay";
 };
 
@@ -170,7 +170,7 @@
 
 &pwm {
 	pinctrl-names = "default";
-	pinctrl-0 = <&pwm1_pins>;
+	pinctrl-0 = <&pwm_pins>;
 	status = "okay";
 };
 
